build-config/resources/Ballerina.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+org = "ballerinax"
+name = "persist.sql"
+version = "@toml.version@"
+authors = ["Ballerina"]
+keywords = ["persist", "sql", "mysql", "experimental"]
+repository = "https://github.com/ballerina-platform/module-ballerinas-persist.sql"
+icon = "icon.png"
+license = ["Apache-2.0"]
+distribution = "2201.6.0"
+
+[[platform.java11.dependency]]
+groupId = "io.ballerinax.persist.sql"
```

Comment:
```suggestion
groupId = "io.ballerina.stdlib"
```

Reply:
Fixed in [c887e05](https://github.com/ballerina-platform/module-ballerinax-persist.sql/pull/1/commits/c887e05a70d6e20c0fcca886464b990a2dd6e1f7)